apps/mobile/src/screens/GroupSettingsWebView/GroupSettingsWebView.js

```diff
@@ -1,27 +1,18 @@
-import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import React, { useCallback } from 'react'
 import { ScrollView, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
 import { useFocusEffect, useNavigation } from '@react-navigation/native'
 import useGroup from '@hylo/hooks/useGroup'
 import useRouteParams from 'hooks/useRouteParams'
-import HyloWebView from 'components/HyloWebView'
+import HyloWebView from 'components/HyloWebView/HyloWebView'
```

Comment:
Does this matter? Importing it directly instead of via its index?

Reply:
Good catch. It won't break anything, but isn't correct. My auto-import in Visual Studio code does this by default and sometimes forget to correct it. Fixed now.